### DISCUSSION TITLE
The "Enable feedback pull requests" option is not working anymore !

### DISCUSSION THREAD

**Discussion by paul-friedli** [community/community#137249] [GitHub Education]
### Hi everyone,

**The "Enable feedback pull requests" option is not working anymore !**

That splendid feature used to work very well and we used it extensively at my school, for tons of projects with our students.

But recently, whatever the project and classroom settings used/tested => classroom seems to totally ignore the "Enable feedback pull requests" option. It doesn't prepare the repo for that.

Once a student accepts the job, the link "Repository" **in classroom** still opens the student repo but **the "Feedback" link shows a "404 not found" page !?!?**

This is very, very problematic with students and anihilates one of the greatest strengths of classroom.

What is going on here ? Am I missing something ??

**Comment by MiguelCardoso19** [comment]
It looks like the "Enable feedback pull requests" feature in GitHub Classroom isn't working as expected. Here’s what you can do:

- Double-check settings: Make sure the option is enabled in your classroom settings.
- Clear browser cache: This can sometimes fix display issues.
- Check GitHub status: See if there are any ongoing issues with GitHub services.
- Contact GitHub Support: If the problem persists, reaching out to them is your best bet for a solution.

This should help get you back on track!

**Reply by paul-friedli** [reply]
Points 1-3 have already been done, several times, and of course **before pretending** there is a problem.

For point 4 (contacting GitHub support), I though I was doing it here.

**Comment by MiguelCardoso19** [comment]
This is the Github community, i can provide you with the right link of the support!

here: https://support.github.com/request

**Reply by paul-friedli** [reply]
When the problem will be solved. Only then. Together with the "solution". Otherwise all this makes no sense to anyone falling on this page.

**Reply by samus-aran** [reply]
Hi @MiguelCardoso19 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by paul-friedli** [reply]
Thank you @samus-aran !

**Current status** : a step forward has been done
- last Friday (30 august) I opened a ticket (#2974957) for this problem.
- **Since Sunday, the "Enable feedback pull request" classroom feature seems to work perfectly again**.

The problem has probably been solved but I'll wait until my ticket gets an official answer, then I'll close this point here with the explanations received.
